tasking/pulp/tasking/constants.py

```diff
@@ -4,6 +4,20 @@
     # The interval in seconds for which the Celery Process monitor thread sleeps between
     # checking for missing Celery processes.
     CELERY_CHECK_INTERVAL=60,
+    # The interval in seconds during which the Celery Processes being monitored need to report a
+    # hearbeat to be considered active.
+    CELERY_TIMEOUT_SECONDS=200,
+    # The interval in seconds with which a secondary resource manager tries to acquire the lock
+    # that is needed to be the primary resource manager.
+    CELERY_TICK_DEFAULT_WAIT_TIME=90,
```

Comment:
Consider renaming this to be `RESOURCE_MANAGER_LOCK_RETRY_ACQUIRE_TIME` or `RESOURCE_MANAGER_LOCK_RETRY_TIME`